source/compatibility.txt

```diff
@@ -0,0 +1,45 @@
+.. _pymongo-compatibility:
+
+=============
+Compatibility
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: backwards compatibility, versions, upgrade 
+
+MongoDB Compatibility
+---------------------
+
+The following compatibility table specifies the recommended version or versions
+of {+driver-short+} for use with a specific version of MongoDB.
+
+The first column lists the driver version.
+
+.. sharedinclude:: dbx/lifecycle-schedule-callout.rst
```

Comment:
Question:
Based on the MongoDB EOL lifecycle statement, is it necessary to show MongoDB versions prior to 3.4 in the compatibility tables?